File: pre-contest/a_billeterie/solutions/a_billeterie.c
#include <stdio.h>
#include <stdlib.h>

int main() {
	int prix2nde, prix1ere, nbBillets2nde, nbBillets1ere;
	scanf("%d %d %d %d", &prix2nde, &prix1ere, &nbBillets2nde, &nbBillets1ere);
	int chiffreAffaire = (prix2nde * nbBillets2nde) + (prix1ere * nbBillets1ere);
	printf("%d\n", chiffreAffaire);
	return EXIT_SUCCESS;
}

File: pre-contest/a_billeterie/solutions/a_billeterie.exs
readInt = fn -> IO.read(:stdio, :line) |> String.trim() |> String.to_integer() end

prix2nde = readInt.()
prix1ere = readInt.()

nbBillets2nde = readInt.()
nbBillets1ere = readInt.()

prixTotal = prix2nde * nbBillets2nde + prix1ere * nbBillets1ere
IO.puts(prixTotal)
